docs/BUILDING_WINDOWS.md

```diff
@@ -0,0 +1,73 @@
+# Building Habitat from source
+
+*NOTE:* This document is likely to change a lot over the coming months.
+
+*NOTE:* For Rust editing, I'd recommend Visual Studio Code and have [a blog post on how to set up the best Rust editing experience](http://stevenmurawski.com/powershell/2016/08/rustify).
```

Comment:
These notes are *very* helpful. A lot of people have their own tool bench but it doesn't have great Rust support just yet